### PR TITLE
docs(gpu):scratch-storage

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Added documentation about scratch storage on H100 Instances

### CODE REVIEW

File: compute/gpu/concepts.mdx [hunk 1]
```diff
@@ -194,8 +194,13 @@ Additionally, it advocates for combining AI with a human touch and common sense
 ## Scratch storage
 
 Scratch storage refers to a type of temporary storage space used in computing systems, particularly in high-performance computing (HPC) environments and supercomputers. 
-It is used for short-term storage of data that is being actively processed, but it is not meant for long-term or permanent storage. 
-It is an ephemeral storage solution that does not support features like snapshots, backups, or restores.
+It is used for short-term cache storage of data that is being actively processed, but it is not meant for long-term or permanent storage.
+Scratch storage is an ephemeral storage solution that does not support features like snapshots, backups, or restores.
+
+<Message type="note">
+  Scratch storage does not survive stopping the server, doing a full stop/start cycle will erase the scratch data. However, doing a simple reboot or using the stop in place function will keep the data.
```
Comment (nerda-codes) [inline compute/gpu/concepts.mdx:201]:
```suggestion
  Scratch storage does not prevent your data from being lost when stopping the server. Doing a full stop/start cycle erases scratch data. However, rebooting or using the stop in place function will keep the data.
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+meta:
+  title: How to use the scratch storage on H100 GPU Instances
```
Comment (nerda-codes) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:3]:
```suggestion
  title: How to use scratch storage on H100 GPU Instances
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  description: This page explains how to use the scratch storage on H100 GPU Instances
```
Comment (nerda-codes) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:4]:
```suggestion
  description: This page explains how to use scratch storage on H100 GPU Instances
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+content:
+  h1: How to use the the scratch storage on H100 GPU Instances
```
Comment (nerda-codes) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:6]:
```suggestion
  h1: How to use scratch storage on H100 GPU Instances
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  paragraph: This page explains how to use the scratch storage on H100 GPU Instances
```
Comment (nerda-codes) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:7]:
```suggestion
  paragraph: This page explains how to use scratch storage on H100 GPU Instances
```

File: compute/gpu/concepts.mdx (hunk 1 above)
Comment (RoRoJ) [inline compute/gpu/concepts.mdx:201]:
```suggestion
  Scratch storage does not survive once the server is stopped: doing a full stop/start cycle will erase the scratch data. However, doing a simple reboot or using the stop in place function will keep the data.
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+dates:
+  validation: 2023-09-18
+  posted: 2023-09-18
+categories:
+  - compute
+---
+
+Scaleway H100 GPU Instances are equipped with additional scratch storage. This form of temporary local storage operates differently from our regular local storage.
+
+Scratch storage temporarily accommodates data during computational or data processing tasks. It finds common use for storing intermediate results, processing input data, or holding output data before it is moved to more permanent storage.
```
Comment (RoRoJ) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:18]:
```suggestion
Scratch storage temporarily accommodates data during computational or data processing tasks. It is commonly used for storing intermediate results, processing input data, or holding output data, before that data is moved to more permanent storage.
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+Unlike conventional storage, scratch storage lacks features like snapshots, backups, or restores. Furthermore, it isn't designed for downloading images; its main function is to serve as a cache. Typically, data in scratch storage is deleted once the computation or processing task concludes.
+
+Scaleway's H100 GPU Instances use NVMe disks for their scratch storage, which are fine-tuned for high-speed data access. This design ensures fast read and write speeds, which is crucial for applications demanding extensive data processing capabilities.
+
+<Message type="note">
+  Scratch storage does not survive stopping the server, doing a full stop/start cycle will erase the scratch data. However, doing a simple reboot or using the stop in place function will keep the data.
```
Comment (RoRoJ) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:25]:
```suggestion
  Scratch storage does not survive once the server is stopped: doing a full stop/start cycle will erase the scratch data. However, doing a simple reboot or using the stop in place function will keep the data.
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+To address this issue, we have implemented scratch storage. Unlike traditional storage, scratch storage does not necessitate the downloading of the entire image (and therefore cannot be used for backup restoration), but it boasts the capability to feed data into the GPU at a significantly accelerated rate. This enhancement allows us to provide the GPU with a substantial amount of scratch storage, ensuring a swifter and more efficient data input process.
+
+<Message type="note">
+      The maximum possible size for scratch storage is
+      * for H100-1-80G Instances: 1920GB
+      * for H100-2-80G Instances: 3940GB
+</Message>
+
+## How to use scratch storage using Scaleway CLI and console
+Scratch storage is automatically added for H100-1-80G and H100-2-80G Instances.
```
Comment (RoRoJ) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:43]:
```suggestion
Scratch storage is automatically added when creating H100-1-80G and H100-2-80G Instances.
```

Comment (RoRoJ) [reply compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:43]: If I have understood this correctly?

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+For example, the GPU Render-S requires several minutes (typically ranging from 6 to 12 minutes) to fetch a 400GB image, a delay that impedes the user's ability to promptly initiate model training. This lag becomes particularly problematic in the realm of AI, where a 400GB dataset is considered relatively small.
+To address this issue, we have implemented scratch storage. Unlike traditional storage, scratch storage does not necessitate the downloading of the entire image (and therefore cannot be used for backup restoration), but it boasts the capability to feed data into the GPU at a significantly accelerated rate. This enhancement allows us to provide the GPU with a substantial amount of scratch storage, ensuring a swifter and more efficient data input process.
+
+<Message type="note">
+      The maximum possible size for scratch storage is
+      * for H100-1-80G Instances: 1920GB
+      * for H100-2-80G Instances: 3940GB
+</Message>
+
+## How to use scratch storage using Scaleway CLI and console
```
Comment (RoRoJ) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:42]:
```suggestion
## How can I add scratch storage to my GPU Instance using the Scaleway CLI or console?
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+Scratch storage is automatically added for H100-1-80G and H100-2-80G Instances.
+
+## How to use scratch storage using the API
```
Comment (RoRoJ) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:45]:
```suggestion
## How can I add add scratch storage to my GPU Instance when using the API?
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+You need to add an extra volume, for example
```
Comment (RoRoJ) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx]: Can this only be done at creation, or can you add it after?

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+"volumes":{"1":{"name":"scratch-volume","volume_type":"scratch","size":1900000000000}}
+
+## How to use scratch storage using Terraform
```
Comment (RoRoJ) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:51]:
```suggestion
## How can I add scratch storage to my GPU Instance using Terraform?
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx (hunk 9 above)
Comment (nerda-codes) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:42]:
```suggestion
## How to use scratch storage using the Scaleway CLI and console?
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx (hunk 10 above)
Comment (nerda-codes) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:45]:
```suggestion
## How to use scratch storage using the API?
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx (hunk 12 above)
Comment (nerda-codes) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:51]:
```suggestion
## How to use scratch storage with Terraform?
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+
+For example, the GPU Render-S requires several minutes (typically ranging from 6 to 12 minutes) to fetch a 400GB image, a delay that impedes the user's ability to promptly initiate model training. 
+This delay becomes particularly problematic in the realm of AI, where a 400GB dataset is considered relatively small.
+
+To address this issue, we have implemented scratch storage. Unlike traditional storage, scratch storage does not necessitate the downloading of the entire image (and therefore cannot be used for backup restoration), but it boasts the capability to feed data into the GPU at a significantly accelerated rate.
+This enhancement allows us to provide the GPU with a substantial amount of scratch storage, ensuring a swifter and more efficient data input process.
+
+<Message type="note">
+      The maximum possible size for scratch storage is
+      * for H100-1-80G Instances: 1920GB
```
Comment (jcirinosclwy) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:38]:
```suggestion
      * for H100-1-80G Instances: 1920 GB
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+      * for H100-2-80G Instances: 3940GB
```
Comment (jcirinosclwy) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:39]:
```suggestion
      * for H100-2-80G Instances: 3940 GB
```

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx [hunk 15]
```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: How to use the scratch storage on H100 GPU Instances
+  description: This page explains how to use the scratch storage on H100 GPU Instances
+content:
+  h1: How to use the the scratch storage on H100 GPU Instances
```
Comment (jcirinosclwy) [inline compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:6]:
```suggestion
  h1: How to use the scratch storage on H100 GPU Instances
```

Comment (jcirinosclwy) [reply compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:6]: @bene2k1 Maybe `How to use scratch storage on H100 GPU Instances`

?

**Review by jcirinosclwy — APPROVED** [review]

File: compute/gpu/how-to/use-scratch-storage-h100-instances.mdx (hunk 9 above)
Comment (jcirinosclwy) [reply compute/gpu/how-to/use-scratch-storage-h100-instances.mdx:42]: @nerda-codes I think it was @RoRoJ who said it sounds weird in English to use a question mark after a `How to` ?
Just to check :)

**Review by RoRoJ — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**